BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h

```diff
@@ -161,18 +162,19 @@ extern BOXRepresentationType *const BOXRepresentationTypeHLS;
 extern BOXRepresentationType *const BOXRepresentationTypeCrocodoc;
 extern BOXRepresentationType *const BOXRepresentationTypeDICOM;
 
+// Representations URL Template
+extern NSString *const BOXRepresentationTemplateKeyAccessPath;
+
+// Representation Template Value
+extern NSString *const BOXRepresentationTemplateValueAccessPath;
```

Comment:
Maybe change this constant name to reference HLS specifically, since we'll have to make this more generic in the future.